src/vellum/workflows/workflows/base.py

```diff
@@ -618,6 +618,17 @@ def validate(cls) -> None:
 
         cls._validate_no_self_edges()
 
+    @classmethod
+    def _get_all_nodes_recursive(cls) -> Iterator[Type[BaseNode]]:
+        """
+        Returns an iterator over all nodes in the Workflow, including nodes nested in subworkflows.
+        """
+        for node in cls.get_all_nodes():
+            yield node
+            subworkflow = node.__dict__.get("subworkflow")
```

Comment:
DevinAI instead of hardcoding the `"subworkflow"` attribute, let's iterate over the node attributes and check if any is subclass'd by `BaseWorkflow`